tests/core/tools/color.js

```diff
@@ -0,0 +1,74 @@
+/* bender-tags: editor */
+/* bender-include: _helpers/colorTools.js */
+/* global colorTools */
+
+
+ // Basic definitions
+ // 3-HEX -> hexadecimal color with only 3 characters value: #FFF
+ // 6-HEX -> hexadecimal color with exactly 6 characters value: #FFFFFF
+ // 8-HEX -> hexadecimal color with exactly 8 characters value: #FFFFFF00. Last two characters are for alpha
+( function() {
+	'use strict';
+
+	bender.test( {
+		setUp: function() {
+			this.createColor = function( colorCode ) {
+				return new CKEDITOR.tools.color( colorCode );
+			};
+		},
+
+		'test color object creation': function() {
+			var colorObject = this.createColor( '' );
+
+			assert.isObject( colorObject );
+		},
+
+		'test color from invalid string color name returns default color': colorTools.testColor( 'NotValidColorName', '#000000', 'getHex' ),
```

Comment:
Second test checks not valid input string with `NotValidColorName` input.

I'll add tests for non-string values and invalid HEX

Reply:
Because of `Array` test case - now I'm wondering if we should create color from array `[ red, green, blue, alpha ]`. But there will be confusion if values are for rgb or hls...